govc/test/vm.bats

```diff
@@ -464,6 +464,19 @@ load test_helper
   assert_failure
 }
 
+@test "vm.migrate" {
+  vcsim_env
+  vm=$(new_empty_vm)
+
+  # migrate from H0 to H1
+  run govc vm.migrate -host DC0_C0/DC0_C0_H1 "$vm"
```

Comment:
Every command using flags.HostSystemFlag gets the hardcoded name "host".  Other examples where host is a "target" include vm.clone, import.ova, vm.markasvm.  The SDK fields/params are also named "host" these cases.
See MigrateVM_Task for example: http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.wssdk.apiref.doc/vim.VirtualMachine.html#migrate